File: Soundmarine.lib/Model/Album.cs
namespace Soundmarine.lib.Model;

public class Album
{
    public string Id { get; set; }
    public string Title { get; set; }
    public string ArtistId { get; set; }

    public Album(string id, string title, string artistId)
    {
        Id = id;
        Title = title;
        ArtistId = artistId;
    }
}

File: soundmarine_client/lib/models/album.dart
class Album {
  final String id;
  final String title;
  final String? artistId;
  final String? artistName;

  Album({required this.id, required this.title, this.artistId, this.artistName});

  factory Album.fromJson(Map<String, dynamic> json) {
    return Album(
      id: json['id'],
      title: json['title'],
      artistId: json['artistId'],
      artistName: json['artistName'],
    );
  }
  
  Map<String, dynamic> toJson() => {
    'id': id,
    'title': title,
    'artistId': artistId,
    'artistName': artistName,
  };
}